packages/component/package.json

```diff
@@ -131,6 +127,8 @@
     "@types/react": "^16.14.60",
     "babel-plugin-istanbul": "^6.1.1",
     "babel-plugin-transform-inline-environment-variables": "^0.4.4",
+    "botframework-webchat-base": "0.0.0-0",
```

Comment:
Double verify, are we sure `-base` and `-styles` should be development dependencies?

Reply:
I like both of them bundled, I just want to verify if the code is good.

Reply:
Otherwise npm won't allow to install packages depending on them. We either have to bundle or publish it seems.

Reply:
Bundle is good, not too worry about code dupe yet, it's better than today.

Also no need to update package dependency is good.